.travis.yml

```diff
@@ -3,22 +3,23 @@ sudo: false
 language: node_js
 
 node_js:
-  - "8"
-  
+  - "9"
+
 notifications:
   disabled: true
-  
+
 before_install:
-  - npm install -g npm@5
+  - npm install -g typescript
```

Comment:
Hi! Drive-by commenter. Shouldn't these just be in devDependencies?

Reply:
Ah, I think you're right! Travis should just use the local typescript in node_modules. I forgot it was already defined in devDependencies.